ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py

```diff
@@ -89,8 +89,8 @@ def print_repairs(repairs, max_lines=-1):
 
 def print_schedule_table(schedule_table, schedules, max_lines):
 
-    sorted_schedules = sorted(schedules, key=lambda sorted_schedule: sorted_schedule.last_repaired_at_in_ms,
-                              reverse=True)
+    sorted_schedules = sorted(schedules, key=lambda x: (x.last_repaired_at_in_ms, x.repaired_ratio, x.status),
```

Comment:
I would prefer if status wasn't part of the sorting. It's a string so it's hard to predict how sorting will be performed.

Reply:
I'm not sure I follow this one. Sorting will be in descending order. The status is somewhat hardcoded throughout, so we couldn't just change it like that anyhow.